api/controllers/v1/massif/get-dashboard-data.js

```diff
@@ -0,0 +1,60 @@
+// const ControllerService = require('../../../services/ControllerService');
+const ErrorService = require('../../../services/ErrorService');
+const DashboardDataMassifService = require('../../../services/DashboardDataMassifService');
+
+module.exports = async (req, res) => {
+  try {
+    // TODO : check if id is present
+    await DashboardDataMassifService.createMaterializedView();
+
+    const caveWithMaxDepth =
+      DashboardDataMassifService.getCaveWithMaxDepthInMassif(req.params.id);
+
+    const nbCaves = DashboardDataMassifService.getNbCavesInMassif(
+      req.params.id
+    );
+
+    const caveWithMaxLength =
+      DashboardDataMassifService.getCaveWithMaxLengthInMassif(req.params.id);
+
+    const nbCavesWhichAreDiving =
+      DashboardDataMassifService.getNbCavesWhichAreDivingInMassif(
+        req.params.id
+      );
+
+    const avgDepthAndLength =
+      DashboardDataMassifService.getAvgDepthAndLengthInMassif(req.params.id);
+
+    const totalLength = DashboardDataMassifService.getTotalLength(
+      req.params.id
+    );
+
+    return Promise.all([
+      nbCaves,
+      caveWithMaxDepth,
+      caveWithMaxLength,
+      nbCavesWhichAreDiving,
+      avgDepthAndLength,
+      totalLength,
+    ]).then((values) => {
+      const data = {
+        nb_caves: values[0].nb_caves,
+        cave_with_max_depth: values[1],
+        cave_with_max_length: values[2],
+        diving_caves: values[3].nb_diving_cave,
+        avg: values[4],
+        total_length: values[5],
+      };
+
+      return res.ok(data);
+    });
+
+    // if (!massif) {
+    //   return res.notFound(`${params.searchedItem} not found`);
+    // }
+
+    // TODO : use ControllerService
```

Comment:
Yes really important TODO here. 
Add you converter `toMassifStatistics` (or `toMassifOverview` as explained in another of my comment) to the `converter.js` file.

Reply:
We've seen what you have done for other entities but is it pertinent in our case, since the statistics we retrieve from the database are not a model or an entity by itself (since it's a gathering of multiple different informations on massif, caves and entrances) ? And if we create a model with sails, is it not going to create a mirror table in PostGres ? 

We don't quite grasp the need to complexify the implementation here...